src/components/IIIFPlayerWrapper.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { useManifestDispatch } from '../context/manifest-context';
 import PropTypes from 'prop-types';
-import { parseManifest } from 'manifesto.js';
+import { parseAutoAdvance } from 'manifesto.js';
```

Comment:
I think this import should be from `iiif-parser` module?

Reply:
Surprised the tests passed with that error!